src/DependencyInjection/DbToolsConfiguration.php

```diff
@@ -73,7 +73,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->children()
                             ->arrayNode('yaml')
                                 ->useAttributeAsKey('connection')
-                                ->beforeNormalization()->ifString()->then(function ($v) { return ['default' => $v]; })->end()
+                                ->beforeNormalization()->ifString()->then(function ($v): array { return ['default' => $v]; })->end()
```

Comment:
Same here, you can remove this type

Reply:
Done

Reply:
The type is still there

Reply:
Fixed https://github.com/makinacorpus/DbToolsBundle/pull/122/commits/cadcd43377fea26c51e0d8171a40189a7b039b8b